pkg/balancer-js/README.md

```diff
@@ -0,0 +1 @@
+# SDK
```

Comment:
Want to take a stab at a readme so we can publish this? I can otherwise write something up myself.

Reply:
Sure, I've popped something in however I'm expecting us to add more to this package before we publicise it so it'll evolve over time.